docs/guide/topics/threads.rst

```diff
@@ -0,0 +1,245 @@
+
+Users with the :attr:`~Permissions.manage_threads` permission can see all threads, including private ones they are not explicitly invited to.
+
+You can view the API reference for creating threads here: :meth:`TextChannel.create_thread` / :meth:`Message.create_thread`.
+
+For a quick primer, let's try making a command that creates a thread and then sends a message in it:
+
+.. code-block:: python3
+
+    @bot.command()
+    async def create_thread(ctx: commands.Context):
+        thread = await ctx.channel.create_thread(name="My cool thread", message=ctx.message)
+
+        await thread.send("Created!")
+
+which will create a thread like so:
+
+.. image:: /images/guide/threads/create_public_thread.png
+
+Simple, right? Let's discuss some of the finer details.
+
+Public threads
+~~~~~~~~~~~~~~~
+
+Public threads can be viewed by anyone who can see the channel they belong to.
+People can browse and join these threads as they please.
+
+All public threads are tied to a specific message. You can use a message that already exists, but you can also send a new message to attach the thread to.
+
+Let's expand on our example from before by making it possible to name the thread. We can also switch to using the convenient :meth:`Message.create_thread` shortcut.
+
+.. code-block:: python3
+    :emphasize-lines: 3
+
+    @bot.command()
+    async def create_thread(ctx: commands.Context, *, thread_name: str):
+        thread = await ctx.message.create_thread(name=thread_name)
+
+        await thread.send(f"Hello from the {thread_name} thread!")
+
+and then we have our public thread:-
+
+.. image:: /images/guide/threads/create_public_thread_param.png
+
+As discussed before, we can see it from the thread list, even if we're not a member:-
+
+.. image:: /images/guide/threads/not_a_member.png
+    :scale: 90
+
+
+Private threads
+~~~~~~~~~~~~~~~~
+
+Private threads exist as "invite only" threads.
+By default, they only contain the person who created them, but other people can be invited to participate.
+
+Private threads can optionally be 'invitable'.
+If a thread is invitable, it means anyone in that thread can invite others to it.
+Otherwise, only the owner and people with the :attr:`~Permissions.manage_threads` permission can add new people to the thread.
+
+Users with :attr:`~Permissions.manage_threads` ("Moderators") can also see a list of private threads they're not in, similar to public threads.
+
+Let's create a private, non-invitable thread with a command:
+
+.. code-block:: python3
+    :emphasize-lines: 5
+
+    @bot.command()
+    async def secret_thread(ctx: commands.Context):
+        thread = await ctx.channel.create_thread(
+            name="Secret Birthday Planning!",
+            invitable=False
+        )
+        await thread.send(ctx.author.mention)
+
+We've chosen here to mention the author as soon as we make the thread.
+If we didn't do this, then the thread would be created, but we wouldn't necessarily be able to see it!
```

Comment:
```suggestion
If we didn't do this, then the thread would be created, but the user, who invoked the command, wouldn't necessarily be able to see it!
```
"We" could mean the bot but the bot would be the only one to see the thread. 
"We" could also mean the developer writing the command however we want the person using the command to see the thread.